Store/Get-LargeMailboxFolderStatistics.ps1

```diff
@@ -0,0 +1,68 @@
+﻿# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
```

Comment:
This might be a good location to make sure that the EXO Module is installed with the correct version.

Reply:
Done